### ISSUE TITLE
Log of discussed papers

### ISSUE THREAD

**Issue by leezu** [ML-HK/paper-discussion-group#12]
For reference, we will collect a list of discussed papers as well as the date of discussion in this issue.

**Comment by leezu** [comment]
**16.02.2017**
[Wasserstein GAN](https://arxiv.org/abs/1701.07875)
(Martin Arjovsky, Soumith Chintala, Léon Bottou)
Submitted to arXiv on 26 Jan 2017
_We introduce a new algorithm named WGAN, an alternative to traditional GAN training. In this new model, we show that we can improve the stability of learning, get rid of problems like mode collapse, and provide meaningful learning curves useful for debugging and hyperparameter searches. Furthermore, we show that the corresponding optimization problem is sound, and provide extensive theoretical work highlighting the deep connections to other distances between distributions._

**Comment by leezu** [comment]
**23.02.2017** [Q-Prop: Sample-Efficient Policy Gradient with An Off-Policy Critic](https://openreview.net/forum?id=SJ3rcZcxl&noteId=SJ3rcZcxl)
(Shixiang Gu, Timothy Lillicrap, Zoubin Ghahramani, Richard E. Turner, Sergey Levine)
ICLR 2017
_Model-free deep reinforcement learning (RL) methods have been successful in a wide variety of simulated domains. However, a major obstacle facing deep RL in the real world is the high sample complexity of such methods. Batch policy gradient methods offer stable learning, but at the cost of high variance, which often requires large batches, while TD-style methods, such as off-policy actor-critic and Q-learning, are more sample-efficient but biased, and often require costly hyperparameter sweeps to stabilize. In this work, we aim to develop methods that combine the stability of policy gradients with the efficiency of off-policy RL. We present Q-Prop, a policy gradient method that uses a Taylor expansion of the off-policy critic as a control variate. Q-Prop is both sample efficient and stable, and effectively combines the benefits of on-policy and off-policy methods. We analyze the connection between Q-Prop and existing model-free algorithms, and use control variate theory to derive two variants of Q-Prop with conservative and aggressive adaptation. We show that conservative Q-Prop provides substantial gains in sample efficiency over trust region policy optimization (TRPO) with generalized advantage estimation (GAE), and improves stability over deep deterministic policy gradient (DDPG), the state-of-the-art on-policy and off-policy methods, on OpenAI Gym's MuJoCo continuous control environments._
